### PR TITLE
refactor/refactor page container

### PR DESCRIPTION
**Pull request by elvisnguy**
## Change Description 👋
• Fix css page container
• Enhance search bar, rows per page.

## Demo 📺
https://www.loom.com/share/1eb6d70185c34fadbaaf8bdef761ab37

### CODE REVIEW

**Comment by truonghungit** [comment]
Please **double-check** again the demo, **TEST** before create a pull request.

https://www.loom.com/share/3ded61eaeca84249b91a4077b5aafdb9

File: src/companies/components/CompaniesTable.vue [hunk 1]
```diff
@@ -59,25 +63,33 @@ export default {
...
       dialogTitle: "",
       dialogContent: "",
+      
     };
   },
 
   methods: {
-    confirmDeleteCompany() {
+    itemRowBackground(item) {
+        return this.companiesDisplay.indexOf(item) % 2 < 1 ? "bg-gray" : "bg-white";
```
Comment (luantc13) [inline src/companies/components/CompaniesTable.vue:76]: `=== 0` is better.

File: src/companies/components/SearchCompanies.vue [hunk 2]
```diff
@@ -7,15 +7,27 @@
...
 <script>
-export default {};
+export default {
+  props:{
+    search:String,
+  },
+  method:{
+    searchChange(value){
+      this.$emit('on-search',value)
+      console.log(value)
```
Comment (luantc13) [inline src/companies/components/SearchCompanies.vue:27]: We should remove all `console.log` before commiting.

File: src/companies/components/SearchCompanies.vue [hunk 3]
```diff
@@ -7,15 +7,27 @@
...
     >
     </v-text-field>
   </div>
 </template>
 
 <script>
-export default {};
+export default {
+  props:{
+    search:String,
```
Comment (luantc13) [inline src/companies/components/SearchCompanies.vue:22]: Not a good formating.

File: src/companies/views/ListCompaniesPage.vue [hunk 4]
```diff
@@ -1,53 +1,81 @@
...
 <script>
 import { mapGetters } from "vuex";
 import store from "@/store";
+import PageContainer from "@/components/PageContainer.vue";
 import CreateButton from "@/components/CreateButton.vue";
 import ShowCompanies from "@/companies/components/ShowCompanies.vue";
 import FilterCompanies from "@/companies/components/FilterCompanies.vue";
 import RowsPerPage from "@/companies/components/RowsPerPage.vue";
-import SearchCompanies from "@/companies/components/SearchCompanies.vue";
+// import SearchCompanies from "@/companies/components/SearchCompanies.vue";
```
Comment (luantc13) [inline src/companies/views/ListCompaniesPage.vue:60]: Pls remove all commented codes if we don't use them anymorre.

File: src/employees/store/index.js [hunk 5]
```diff
@@ -68,13 +68,11 @@ const actions = {
...
-        value: employeeStatus,
-        path: "/status/",
-        op: "replace",
-      },
-    ];
+    let payload = [{
+      value: employeeStatus,
+      path: "/status/",
+      op: "replace",
+    }, ];
```
Comment (luantc13) [inline src/employees/store/index.js:75]: `payload` should be a `const`.

Btw, I prefer the old format.

File: src/employees/store/index.js [hunk 6]
```diff
@@ -131,4 +129,4 @@ export default {
   mutations,
   actions,
   getters,
-};
+};
```
Comment (luantc13) [inline src/employees/store/index.js:132]: All files should have an empty line at the end.

File: src/layouts/dashboard/AppBar.vue [hunk 7]
```diff
@@ -107,10 +109,25 @@ export default {
         this.$route.push({ path: "/auth/signin" });
       });
     },
+      navigateCompanies() {
+      this.$router.push({
+        path: `/companies`,
+      });
+    },
```
Comment (luantc13) [inline src/layouts/dashboard/AppBar.vue:116]: I think we have a bad formating here.

Comment (luantc13) [reply src/layouts/dashboard/AppBar.vue:116]: This method name should be `navigateToCompanieListPage`.

File: src/products/views/CreateProduct.vue [hunk 8]
```diff
@@ -11,13 +14,15 @@
...
+    </template>
+  </PageContainer>
 </template>
 
 <script>
 import CreateProductForm from "../components/CreateProductForm.vue";
+import PageContainer from "@/components/PageContainer.vue";
 export default {
-  components: { CreateProductForm },
+  components: { CreateProductForm,PageContainer },
```
Comment (luantc13) [inline src/products/views/CreateProduct.vue:25]: Bad formating.

File: src/products/views/ProductsDashboard.vue [hunk 9]
```diff
@@ -70,10 +77,11 @@ import { mdiMagnify, mdiCogOutline } from "@mdi/js";
 import { mapGetters } from "vuex";
 import store from "@/store";
 import Products from "@/products/components/Products.vue";
+import PageContainer from "@/components/PageContainer.vue";
 import CreateButton from "@/components/CreateButton.vue";
 import { ENABLE_STATUS, DISABLE_STATUS } from "@/shared/variables/index";
 export default {
-  components: { Products, CreateButton },
+  components: { Products, CreateButton ,PageContainer},
```
Comment (luantc13) [inline src/products/views/ProductsDashboard.vue:84]: The same here.

File: src/router/index.js [hunk 10]
```diff
@@ -207,5 +220,6 @@ router.beforeEach((routeTo, routeFrom, next) => {
   } else {
     next();
   }
+
 });
-export default router;
+export default router;
```
Comment (luantc13) [inline src/router/index.js:225]: All files should have an empty line at the end.

File: src/store/index.js [hunk 11]
```diff
@@ -24,7 +24,6 @@ export default new Vuex.Store({
     employees,
     forgotPassword,
     resetPassword,
-    product,
     products,
   },
-});
+});
```
Comment (luantc13) [inline src/store/index.js:29]: All files should have an empty line at the end.

**Review by luantc13 — CHANGES_REQUESTED** [review]
Nice work!
Still having some comments, we can merge after resolving all of them.

File: src/employees/store/index.js (hunk 5 above)
Comment (elvisnguy) [reply src/employees/store/index.js:75]: This "Employees" module did by Duyet, so we're discuss about this and resolve tomorrow

File: src/products/views/CreateProduct.vue (hunk 8 above)
Comment (elvisnguy) [reply src/products/views/CreateProduct.vue:25]: Like above problem, Ngoc did this module so I'll discuss with her and fix on tomorrow morning

File: src/products/views/ProductsDashboard.vue (hunk 9 above)
Comment (elvisnguy) [reply src/products/views/ProductsDashboard.vue:84]: Yeah, I got it.

File: src/router/index.js (hunk 10 above)
Comment (elvisnguy) [reply src/router/index.js:225]: Let me setting or download new formatter later, I try to setting so many times but it can't working

**Review by luantc13 — APPROVED** [review]
Nice work!

**Review by truonghungit — APPROVED** [review]

**Pull request merged**